### PR TITLE
gtk3, enable 32bit

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
In preparation for GIMP on 32bit ... (builds OK)

**Comment by Begasus** [comment]
As mentioned in https://github.com/haikuports/haikuports/issues/6547 GIMP builds ok but fails to launch, doing a check for inkscape (which needs gtkmm3) atm (32bit)

File: x11-libs/gtk3/gtk3-3.24.30.recipe [hunk 1]
```diff
@@ -5,14 +5,14 @@ ranging from small one-off projects to complete application suites."
 HOMEPAGE="https://www.gtk.org"
 COPYRIGHT="GNOME Desktop"
 LICENSE="GNU LGPL v2"
-REVISION="5"
+REVISION="6"
```
Comment (waddlesplash) [inline x11-libs/gtk3/gtk3-3.24.30.recipe:8]: Not needed.

Comment (waddlesplash) [reply x11-libs/gtk3/gtk3-3.24.30.recipe:8]: Bumping the revision will cause rebuilds everywhere. Merely enabling the architecture suffices.

Comment (Begasus) [reply x11-libs/gtk3/gtk3-3.24.30.recipe]: Done

**Pull request merged**